test/jdk/security/infra/java/security/cert/CertPathValidator/certification/SSLCA.java

```diff
@@ -47,6 +47,7 @@
 
     public static void main(String[] args) throws Exception {
 
+        System.setProperty("jdk.security.certpath.ocspNonce", "true");
```

Comment:
Trying to understand this change - why do we need this change and only in couple of tests? Did these test fail with your change?

Reply:
No, these test did not fail with my changes. These two tests are being changed to set the system property jdk.security.certpath.ocspNonce=true in order to send the nonce as the responders in these tests will take the nonce and return it their responses, so we could test the benefit of having nonce extension set.